src/rdf4cpp/Node.cpp

```diff
@@ -243,3 +243,10 @@ Literal Node::as_ebv(storage::DynNodeStoragePtr node_storage) const noexcept {
 }
 
 }  // namespace rdf4cpp
+
+auto std::formatter<rdf4cpp::Node>::format(rdf4cpp::Node n, format_context &ctx) const -> decltype(std::formatter<std::string_view>::format(std::string_view{}, ctx)) {
+    auto s = rdf4cpp::writer::StringWriter::oneshot([n](rdf4cpp::writer::StringWriter &w) {
```

Comment:
For the sake of performance, you should write a `BufOStreamWriter`-like type (with a stack buffer) for `std::formatter` because that avoids a heap allocation